website/content/docs/setup/istio/index.md

```diff
@@ -1,11 +1,32 @@
 ---
-title: Istio security policies deployment
-linktitle: Istio security policies deployment
+title: Istio post-installation steps
+linktitle: Istio post-installation steps
 weight: 30
 ---
 
+## Turn on MutatingWebhook
+
+Upon {{< name >}} deployment the Envoy sidecar is automatically injected to pods in the control-plane namespace (called 4d-system be default). To support automatic injection Istio's [MutatingAdmissionWebhook](https://istio.io/latest/docs/setup/additional-setup/sidecar-injection/#automatic-sidecar-injection) should be turned on in the cluster.
+
+To check if MutatingAdmissionWebhook is turned on in the cluster run:
+
+```bash
+kube-apiserver -h | grep enable-admission-plugins
+```
+
+To turn it on in Kind run:
```

Comment:
I don't think that this is needed. I used istio with kind before with auto injection enabled and never had to use kubeadm.

Reply:
OK, will remove this sentence then.